consul/hooks/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -xe
+
+exec 2>&1
+
+exec consul agent -dev
```

Comment:
Would a `pkg_svc_run="consul agent -dev"` in the plan work? Then the run hook gets generated from that.